docs/Installation-Guide.rst

```diff
@@ -179,26 +224,25 @@ Then run the following commands:
 
 **Note**: glibc >= 2.14 is required.
 
-OSX
-^^^
-
-Install **gcc** and **Open MPI** first:
+macOS
+^^^^^
+See instruction above to install **gcc** and **CMake**
 
 .. code::
 
-  brew install openmpi
-  brew install cmake
-  brew install gcc --without-multilib
+   brew install openmpi
+   brew install cmake
+   brew install gcc --without-multilib
```

Comment:
Why do we need brew now and not before?

Reply:
For open MPI there a bug right now with the lates macOS to install directly.
the standard :`./configure`, `make`; `sudo make install` no longer work.

I didn't install the MPI version, will wait for the bug fix.

Reply:
Here is the issue for the openMPI problem referred to
https://github.com/open-mpi/ompi/issues/4373#issuecomment-338695218

And the fix is to use clang to compile... I have tried and it's working